ledger-core/v2/reference/global_test.go

```diff
@@ -42,17 +42,65 @@ func randLocal() Local {
 	return id
 }
 
+func flipHeader(b []byte) []byte {
+	// Encoder uses BigEndian, while AsBytes uses LittleEndian for the header
+	// b[0], b[1], b[2], b[3] = b[3], b[2], b[1], b[0]
+	return b
```

Comment:
something is missing over here or function is not needed anymore

Reply:
This is stashed as first I'd like to merge this branch, and then fix the encoding.